iroha-lib/model/Query.hpp

```diff
@@ -20,10 +20,9 @@ class Query {
     QueryGenerator query_generator_;
 
 public:
-    Query(
-            const iroha::keypair_t& keypair,
-            uint64_t counter = 0u,
-            const uint64_t created_time = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count());
+    Query(const iroha::keypair_t& keypair,
+          uint64_t counter = 1u,
+          const uint64_t created_time = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count());
```

Comment:
```suggestion
          uint64_t created_time = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count());
```

This is a hot discussion, but as far as I remember this does nothing in function declaration if variable is passed by value. So it just complicates the reading of signature

Reply:
You are indeed correct.

Reply:
I removed the `const` here:)